Owlcat/FitsTool.py

```diff
@@ -295,19 +300,17 @@ def main():
     if not imagenames:
         parser.error("No images specified. Use '-h' for help.")
 
-    # print "%d input image(s): %s"%(len(imagenames),", ".join(imagenames))
-    images = [pyfits.open(img) for img in imagenames]
-    updated = False
-
     # Stack FITS images
     if options.stack:
         if len(imagenames) < 1:
             parser.error("Need more than one image to stack")
-        stack_args = options.stack.split(":")
-        if len(stack_args) != 2:
-            parser.error("Two --stack options are required. See ./fitstool.py -h")
+        if ':' in options.stack:
+            outfile, axis = options.stack.split(":", 1)
```

Comment:
Think this needs a check for ambiguity - -o should not be specified if the OUTFILE:AXIS syntax is used?